docs/dev/reference/hooks/prepareFormData.md

```diff
@@ -35,7 +35,42 @@ distribution or data storage.
 
 
 ## Example
+{{< tabs groupId="prepareFormData-example" >}}
+{{% tab name="Since Contao 5.2" %}}
+```php
+// src/EventListener/PrepareFormDataListener.php
+namespace App\EventListener;
 
+use Contao\CoreBundle\DependencyInjection\Attribute\AsHook;
+use Contao\Form;
+
+#[AsHook('prepareFormData')]
+class PrepareFormDataListener
+{
+    public function __invoke(
+        array &$submittedData, 
+        array $labels, 
+        array $fields, 
+        Form $form, 
+        array $files
```

Comment:
```suggestion
        array &$files
```

I think this needs to be passed by reference, otherwise the code below won't really do anything, will it?

Reply:
Checkt my latest code and you're right :)